tests/unit/test_listing.py

```diff
@@ -38,20 +39,43 @@ def _tree_to_entries(tree: dict, path=""):
 @pytest.fixture
 def listing(test_session):
     catalog = test_session.catalog
-    dataset_name, _, _, _ = DataChain.parse_uri("s3://whatever", test_session)
+    dataset_name, _, _, _ = DataChain.parse_uri("file:///whatever", test_session)
```

Comment:
[C]: minor semi-related change - there is not reason to use `s3`, tests were failing since it was hitting actual token update w/o proper fixtures. cc @ilongin